src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -0,0 +1,425 @@
+---
+title: 'Advanced Data Scrubbing (Preview)'
+sidebar_order: 1
+keywords: ["pii", "gdpr", "personally identifiable data", "compliance"]
+---
+
+In addition to using [`beforeSend`]({% link _documentation/data-management/sensitive-data.md %}#custom-event-processing-in-the-sdk) in your SDK or our [server-side data scrubbing features]({% link _documentation/data-management/sensitive-data.md %}#server-side-scrubbing) to redact sensitive data, we are currently beta-testing ways to give you more granular control over what exactly our server-side data scrubbing does to your events. Additional functionality includes:
+
+* Ability to define custom regular expressions to match on sensitive data
+* More fine-grained tuning on which parts of an event to scrub
+* Partial removal or hashing of sensitive data instead of just deletion
+
+## Overview
+
+**Advanced Data Scrubbing is currently only available if your organization is an Early Adopter.** To become one, go to your organization settings and enable the "Early Adopter" option to get access to new features before they're fully released. This option can be just as easily disabled as it can be enabled.
+
+You will then get access to one new option in both organization settings and the settings of each project called "Advanced datascrubber configuration". Its content is a piece of JSON-based configuration in the format described in the next section.
+
+Note that everything you configure there will have direct impact on all new events, just as all the other data privacy-related settings do. However, it is not possible to break or undo any other data privacy settings that you may have configured. In other words, it is only possible to accidentally remove too much data, not too little.
+
+If you have any questions related to this feature, feel free to contact us at `markus@sentry.io`.
+
+## A basic example
+
+Say you have an exception message which, unfortunately, contains IP addresses which are not supposed to be there. You'd write:
```

Comment:
Suggest "One example is an exception message that includes an IP address that needs to be removed. In this example, you'd write:"